src/ORM/Table.php

```diff
@@ -2304,4 +2305,28 @@ public function __debugInfo()
             'connectionName' => $conn ? $conn->configName() : null
         ];
     }
+
+    /**
+     * Validates that a belongsToMany or hasMany assoc has at least one entry.
+     *
+     * This can be used to validate any other array as well.
+     *
+     * @param mixed $value
+     * @return boolean
+     */
+    public function validateCount($value, $expectedCount = 0, $operator = '>')
```

Comment:
Well, by looking at the rules, why is Table::validateUnique() not a rule? It looks like it could be implemented as such.